vnext/Desktop.DLL/react-native-win32.x64.def

```diff
@@ -5,12 +5,9 @@
 ; **************************************************************************************************
 
 EXPORTS
-??0dynamic@folly@@QEAA@$$QEAU01@@Z
```

Comment:
@acoates-ms Does this look right?

Reply:
I think they may have deleted these methods.  I wouldn't block the update of folly on these anyway.  -- If we need to readd some other methods here for Office we'll get a better sense when Office actually takes in the change.